### ISSUE TITLE
Clean up package dependencies

### ISSUE THREAD

**Issue by JordiBolibar** [ODINN-SciML/ODINN.jl#94]
We need to double check that all the packages we're using are still relevant. This slows down precompilation time a lot.

**Comment by JordiBolibar** [comment]
I recycled this script in order to track unused package dependencies:

```julia
using ODINN
using Pkg.TOML
deps = TOML.parsefile("Project.toml")["deps"]
remove = String[]
for d in keys(deps)
    if !isdefined(ODINN, Symbol(d))
        push!(remove, d)
    end
end
println("rm ", join(remove, " "))
```

**Comment by JordiBolibar** [comment]
Implemented in #107
